### DISCUSSION TITLE
LBT or FCC?

### DISCUSSION THREAD

**Discussion by inventor7777** [pascallanger/DIY-Multiprotocol-TX-Module#994] [Q&A] [answered]
Hi!!

I know that RadioMaster has LBT (listen before talk...?) and FCC versions of the 4-in-1  radios. I read online that there is no hardware difference, just software.

If true, how do I know which I have on my 4-in-1 TX16S Max?

Thanks!

**Comment by pascallanger** [comment]
It's only software related. The module can run both fw. Pick and install the one you want/need.

**Reply by inventor7777** [reply]
How do I know which firmware my internal multi module is running?

**Reply by pascallanger** [reply, marked as answer]
LBT only has a few protocols, so it's quite easy to see. If you have a long list it's FCC. FCC includes LBT or other words LBT is a subset of FCC/a protocol selection of what is in FCC.

**Reply by inventor7777** [reply]
Ohhh okay, that makes sense! Thank you very much :)
